src/components/Modal/modal.css

```diff
@@ -0,0 +1,10 @@
+.modal-backdrop {
+    position: fixed;
+    top: 50%;
+    left: 50%;
+    width: 100%;
+    height: 100%;
```

Comment:
suggestion: add HTML title attribute to buttons to make it easier for users to understand what a button does

Reply:
Will add it